File: cpp/p006.cpp
#include <cmath>
#include <iostream>

int compute(const int LIMIT) {
	int difference = 0;
	difference = pow(LIMIT * (LIMIT + 1) / 2, 2) - (LIMIT * (LIMIT + 1) * ((2 * LIMIT) + 1) / 6);
	return difference;
}

int main() {
	std::cout << compute(100) << std::endl;
	return 0;
}

File: python/p006.py
import math

def compute(LIMIT):
	difference = int((math.pow(LIMIT * (LIMIT + 1) // 2, 2)) - (LIMIT * (LIMIT + 1) * ((2 * LIMIT) + 1) // 6))
	return difference

if __name__ == "__main__":
	print(compute(100))
